dev/com.ibm.websphere.javaee.jaxb.2.3/bnd.bnd

```diff
@@ -0,0 +1,47 @@
+#*******************************************************************************
+# Copyright (c) 2017 IBM Corporation and others.
```

Comment:
2018

Reply:
ahh, I focused on the java files, I'll clean these up.